package/network/ipv6/odhcp6c/files/dhcpv6.sh

```diff
@@ -96,13 +96,23 @@ proto_dhcpv6_setup() {
 
 	[ "$verbose" = "1" ] && append opts "-v"
 
+	json_for_each_item proto_dhcpv6_add_sendopts sendopts opts
+
+	# Dynamically add OROs to support loaded packages.
+	json_load "$(ubus call network get_proto_handlers)"
+	json_get_vars handler_map map
+	json_get_vars handler_dslite dslite
```

Comment:
This is not working like this. With this fix it actually reads the vars:
```suggestion
	json_get_var handler_map map
	json_get_var handler_dslite dslite
```